storage/pruning/pruningStorer.go

```diff
@@ -201,8 +201,12 @@ func initPersistersInEpoch(
 	if oldestEpochActive < 0 {
 		oldestEpochActive = 0
 	}
+	if !args.CleanOldEpochsData {
+		oldestEpochKeep = 0
```

Comment:
This solution is not quite extensible. If we have thousands of epochs, the node will soon run out of memory. Plus, as seen today opening a storage is a quite costly operation...on the order of magnitude of seconds.

Reply:
Sketched a solution.